server/__init__.py

```diff
@@ -0,0 +1,21 @@
+from flask import Flask
+from server.constants import CONSTANTS
+from server.controllers.abc import DatabaseDriver
+from server.controller.mongo import MongoDriver
+import logging
+
+logging.basicConfig(level=logging.DEBUG)
+
+app = Flask(__name__, static_folder='build')
+db: DatabaseDriver = None
+
+if CONSTANTS['ENV'] == 'development':
+    logging.debug('Development environment, using Mongo driver.')
+    logging.warning('Do not use this in production')
```

Comment:
These should be together or at least at the same level. Cause if you're filtering logs for warning or above you would just see
```
Do not use this in production
```
Which would be a very cryptic warning

Reply:
fair point!